components/butterfly/src/rumor.rs

```diff
@@ -333,6 +388,46 @@ mod storage {
             }
             result
         }
+
+        pub fn keys_rsr(&self) -> Vec<RumorKey> {
```

Comment:
I think we can eliminate this function. There's [only one place it's called](https://github.com/habitat-sh/habitat/blob/989e262d51aef6026710c96c5978f8bb002379f1/components/butterfly/src/server.rs#L643), and that caller can call `lock_rsr().rumors().map(RumorKey::from)` itself.

Reply:
Ok